protobuf/src/main/java/com/linecorp/armeria/server/protobuf/ProtobufResponseConverterFunction.java

```diff
@@ -128,6 +133,23 @@ public HttpResponse convertResponse(ServiceRequestContext ctx, ResponseHeaders h
             }
             return HttpResponse.of(headers, toJsonHttpData(result, charset), trailers);
         } else {
+            if (isJsonSeq(contentType)) {
+                checkArgument(result != null, "a null value is not allowed for %s", contentType);
+                if (result instanceof Publisher) {
+                    @SuppressWarnings("unchecked")
+                    final Publisher<Object> publisher = (Publisher<Object>) result;
+                    return JsonTextSequences.fromPublisher(headers, publisher, trailers, this::toJson);
```

Comment:
Doesn't this mean a user can easily produce a malformed JSON element? I think we should not design our API to allow a user to write an arbitrary string that can violate the specification.

Reply:
But I'm not sure I have a better idea at the moment. :sweat_smile:

Reply:
> Doesn't this mean a user can easily produce a malformed JSON element?

Yeah. This API is too open to users and it does not validate whether the content has a correct JSON encoding.

> But I'm not sure I have a better idea at the moment. 😅 Maybe hiding it from the public API?

That makes sense. Let me remove this from public API and use reflection for calling it.